src/test/java/edu/ucsb/cs56/mapache_search/HomePageTest.java

```diff
@@ -66,11 +66,33 @@ public void getHomePage_ContentType() throws Exception {
                 .andExpect(content().contentType("text/html;charset=UTF-8"));
     }
 
+    @Test
+    public void getHomePage_hasCorrectTitle() throws Exception {
+        mvc.perform(MockMvcRequestBuilders.get("/").accept(MediaType.TEXT_HTML)).andExpect(status().isOk())
+                .andExpect(xpath("//title").exists())
+                .andExpect(xpath("//title").string("Mapache Search - It's Spanish for \"raccoon\""));
```

Comment:
Doesn't `.string()` imply `.exists()`? Maybe we can simplify this to just the `.string()` check.

Reply:
Good point! Done.